### PR TITLE
US142491 Add create forum action

### PR DESCRIPTION
**Pull request by dustxd**
### Summary of Changes
* Added `forumsHref` to `DiscussionForumEntity` (individual)
* Added `DiscussionForumsEntity` (collection)
* Added `create-forum` action

### Rally Story
[US142491](https://rally1.rallydev.com/#/?detail=/userstory/644998737805&fdp=true): [ui] Create change forum dialog

### CODE REVIEW

File: src/activities/discussions/DiscussionForumEntity.js [hunk 1]
```diff
@@ -1,11 +1,21 @@
...
 
 /**
  * DiscussionForumEntity class representation of a D2L Discussion Forum.
  */
 export class DiscussionForumEntity extends Entity {
+	/**
+	 * @returns {string} the href of the collection that the discussion forum belongs to
+	 */
+	forumsHref() {
+		if (!this._entity || !this._entity.hasLinkByRel(Rels.Discussions.forums)) {
```
Comment (mostlyfabulous) [inline src/activities/discussions/DiscussionForumEntity.js:13]: Maybe I have to change the rel for this to work over here: https://github.com/Brightspace/lms/pull/28852#discussion_r996115944

Comment (dustxd) [reply src/activities/discussions/DiscussionForumEntity.js:13]: Yeah was gonna ask you what the rel should be here

Comment (mostlyfabulous) [reply src/activities/discussions/DiscussionForumEntity.js:13]: It is serialized like this: image

Comment (mostlyfabulous) [reply src/activities/discussions/DiscussionForumEntity.js:13]: I've added the rel to match your siren entity!

**Review by mostlyfabulous — APPROVED** [review]

File: src/activities/discussions/DiscussionForumEntity.js (hunk 1 above)
Comment (dustxd) [reply src/activities/discussions/DiscussionForumEntity.js:13]: Thank you so much! 😄

**Pull request merged**